cdap-cli/src/main/java/co/cask/cdap/cli/util/FilePathResolver.java

```diff
@@ -52,14 +52,18 @@ public FilePathResolver() {
   /**
    * Resolves a bash-style file path into a {@link File}.
    *
-   * Handles ".", "..", and "~".
+   * Handles ".", "..", "~", and escaped space.
    *
    * @param path bash-style path
    * @return {@link File} of the resolved path
    */
   public File resolvePathToFile(String path) {
     path = resolveVariables(path);
 
+    if (path.contains("\\ ")) {
+      path = path.replace("\\ ", " ");
+    }
+
```

Comment:
but I guess the auto-complete should be compatible with what the CLI understands.